### ISSUE TITLE
Explicitly declaring value of "aria-hidden" in sidebar component not working

### ISSUE THREAD

**Issue by xianx** [hunvreus/basecoat#51]
explicitly declaring 'aria-hidden' value in sidebar component not working.

```
<aside class="sidebar" aria-hidden="<true|false>">
```

sidebar.js overwrites whatever aria-hidden value you put in the HTML. So it cannot be possible to control the default state of the sidebar.

**Comment by xianx** [comment]
setting aria-hidden="false" data-initial-open="false" is working.

closing for now.
